databricks/notebooks/rapids_airline_hyperopt.ipynb

```diff
@@ -0,0 +1,419 @@
+    "      \n",
+    "### MLFlow\n",
+    "\n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[MLflow](https://mlflow.org/) is used for managing the machine learning lifecycle. It provides a way to track experiements and store data about them. MLflow also provides a way to deploy ML models. We'll make use of MLflow to store information about the models and use the built-in integration of MLflow into Databricks to register and store models.\n",
+    "\n",
+    "### Hyperopt\n",
+    "\n",
+    "[Hyperopt](http://hyperopt.github.io/hyperopt/) is a library for finding the best hyperparameters for a given objective function. It provides a way to choose the objective function, the search algorithm, the database in which to store the eval points. We'll use this to define the parameter space to search over. \n",
+    "\n",
+    "We'll use airlines data for this demo - the aim is to predict if an airline will be delayed or not using a target variable `ArrDelayBinary`. We'll use a Random Forest Classifier as the model for the learning task. \n",
+    "\n",
+    "We will compare the performance between the CPU version with scikit-learn to the GPU version with RAPIDS libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\", category=DeprecationWarning)\n",
+    "warnings.filterwarnings(\"ignore\", category=FutureWarning)\n",
+    "\n",
+    "import cudf\n",
+    "import cuml\n",
+    "\n",
+    "import mlflow\n",
+    "import hyperopt\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import mlflow.sklearn\n",
+    "from mlflow.tracking.client import MlflowClient\n",
+    "from hyperopt import fmin, tpe, hp, Trials, STATUS_OK\n",
+    "\n",
+    "import cuml.ensemble\n",
+    "import cuml.metrics\n",
+    "import cuml.preprocessing.model_selection\n",
+    "\n",
+    "import sklearn.ensemble\n",
+    "import sklearn.metrics\n",
+    "import sklearn.model_selection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preapring the data\n",
+    "\n",
+    "1. Download the data into your local system from [Airline 20M](https://rapids-csp.s3-us-west-2.amazonaws.com/data/airline_20000000.orc). This file is in ORC format, since Pandas can't easily read ORC files, we will convert the file to Parquet. \n",
```

Comment:
Could we just share the .parquet file from this bucket?
Also, can people execute a python command to pull the data onto dbfs without having to go through a local system?

Reply:
Maybe just use urllib to grab from that bucket?

Reply:
The bucket doesn't have the parquet version of this file. And I don't have permissions to upload it. Do I ask Miro to do it?